### PR TITLE
[flang] Document DEFAULT_SYSROOT usage on Darwin

### PR DESCRIPTION
**Pull request by luporl**

### CODE REVIEW

**Review by kkwli — APPROVED** [review]
LG. Thanks.

File: flang/docs/GettingStarted.md [hunk 1]
```diff
@@ -88,6 +88,10 @@ cmake \
 ninja
 ```
 
+On Darwin, to make flang able to link binaries with the default sysroot without
+having to specify additional flags, use the `DEFAULT_SYSROOT` cmake flag, e.g.
```
Comment (banach-space) [inline flang/docs/GettingStarted.md:92]:
```suggestion
having to specify additional flags, use the `DEFAULT_SYSROOT` CMake flag, e.g.
```

**Review by banach-space — APPROVED** [review]
LGTM, thanks!

File: flang/docs/GettingStarted.md (hunk 1 above)
Comment (luporl) [reply flang/docs/GettingStarted.md:92]: I missed this one. Fixed on a follow up commit.

**Pull request merged**